src/js/components/Notification/__tests__/__snapshots__/Notification-test.js.snap

```diff
@@ -172,6 +172,7 @@ exports[`Notification custom theme 1`] = `
           <svg
             aria-label="Status is critical"
             class="c4"
+            height="medium"
```

Comment:
Assuming this will stop leaking through after releasing grommet-icons?

Reply:
Correct, we will need to update grommet master snapshots once grommet-icons is released and this will be removed.